packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -184,7 +184,7 @@ $theme-summary-box-link-color: default !default;
 $theme-summary-box-text-color: default !default;
 
 // Table
-$theme-table-border-color: default !default;
+$theme-table-border-color: "ink" !default;
```

Comment:
`$theme-table-border-color` needs a defined token value or it will throw an error. `ink` is the current rendered default value.